pkg/plugins/trivy/plugin.go

```diff
@@ -213,26 +213,55 @@ func (p *plugin) NewConfigForConfigAudit(ctx trivyoperator.PluginContext) (confi
 	return getConfig(ctx)
 }
 
-func (p *plugin) parseImageRef(imageRef, imageDigest string) (v1alpha1.Registry, v1alpha1.Artifact, error) {
-	ref, err := containerimage.ParseReference(imageRef)
+func ParseImageRef(imageRef string, imageDigest string) (v1alpha1.Registry, v1alpha1.Artifact, error) {
+	parts := strings.Split(imageRef, "@")
+	namePart := parts[0]
+	var hasDigest bool
+	var hasTag bool
```

Comment:
do we really need these variables?
`artifact.Digest == ""` means `hasDigest is false`, right?